Documentation/UploadBinaryData.md

```diff
@@ -48,58 +83,76 @@ need to be followed:
      * Spacing should be different from ones, and it should be anisotropic.
      * Direction should be different from identity.
 
-Using Girder to get the SHA512 hash
------------------------------------
+Upload new testing data
+-----------------------
 
 ### Prerequisites
 
-The [data.kitware.com] server is an ITK community resource where any community
-member can upload binary data files. There are three methods available to
-upload data files:
+[web3.storage] is a decentralized IPFS storage
+provider where any ITK community member can upload binary data files.
+There are two methods available to upload data files:
+
+1.  The CMake ExternalData Web3 upload browser interface.
+2.  The <span class="title-ref">w3</span> command line executable that
+    comes with the [@web3-storage/w3] Node.js NPM package.
+
+Once files have been uploaded to your account, they will be publicly
+available and accessible since data is content addressed on the IPFS
+peer-to-peer network. At release time, the release manager will upload
+and archive repository data references in other redundant storage
+locations.
+
+### Upload Via the Web Interface
+
+Use the [CMake ExternalData Web3
+Upload](https://cmake-w3-externaldata-upload.on.fleek.co/) tool to
+upload your data to the InterPlanetary Filesystem and download the
```

Comment:
Shorten to IPFS?

Reply:
We now have shorten links:

- https://content-link-upload.itk.org
- https://content-link-upload.itk.eth.limo

Reply:
Sorry Matt, I meant shortening "InterPlanetary Filesystem", as it has been defined before, and it is used in its shortened form in other appearances.

Reply:
Ah, changed.